### ISSUE TITLE
Z Seam at overhangs

### ISSUE THREAD

**Issue by ursucristian** [Ultimaker/Cura#12220]
### Is your feature request related to a problem?

Is there a way to prevent starting a layer on overhangs? Other than setting your seam manually opposite to the overhanging part.
But what can you do when you have a complicated model with overhangs all over the place?
I have been busy for a while testing on small parts for printing the very popular articulated dragon (https://cults3d.com/en/3d-model/game/articulated-dragon-mcgybeer) and I still have problems with the overhangs of the "horns". I want to make the print as nice as possible thus I must go slow, but at the horns I need to increase the speed so that I get no lifting (at the overhanging part. It looks like I have my settings tuned ok at the moment but I believe the quality would be much better if the seam wouldn't be (sometimes) on the overhang.
![Screenshot1](https://user-images.githubusercontent.com/105577759/168651780-07175591-8486-449f-b7b5-9d35eb089de8.jpg)
![Screenshot2](https://user-images.githubusercontent.com/105577759/168651783-5acaa184-76b3-4759-b330-2ead0af1c366.jpg)

### Describe the solution you'd like

How about a box we can tick to prevent starting on overhangs?

### Describe alternatives you've considered

Manual definition of the location of the seam. But it doesn't work when the overhangs are all around the model.

### Affected users and/or printers

Everybody

**Comment by smartavionics** [comment]
You may care to try my Cura variant that already has this capability. You can find my builds at https://github.com/smartavionics/Cura/releases.

**Comment by PapsOu** [comment]
Hello, and what about supporting that feature in the official Cura ?

I think it is a really important feature to avoid starting lines on top of nothing, like that case I have now : 
![image](https://user-images.githubusercontent.com/5792207/202434133-d55cd1a2-04e3-4c19-9719-5620b5ba603e.png)

**Comment by luizbgomide** [comment]
I agree that this is a very desirable feature. In some parts it is impossible to avoid this, even when playing with Seam placement, which ends up in overhangs that curls up or get deformed due to the excess heat.

**Comment by DejavuDmitry** [comment]
Yeah, also have this problem

**Comment by Flopalop2** [comment]
The new overhang settings prevents this, but would like more flexibility. Right now the seam is placed at the start of the area it deems an overhang so its at a point where flow is changed drastically if you print overhangs slower. Which messes up the seam and overhang.

**Comment by falkenhawk** [comment]
> The new overhang settings prevents this

it does not for me. Cura 5.6.0
![image](https://github.com/Ultimaker/Cura/assets/14310995/5dcd8acd-fd36-496c-a5bf-44b24dc2edb6)
